android/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Happy News</string>
+    <string name="title_settings">Instellingen</string>
+    <string name="title_settings_about">Over</string>
+    <string name="title_settings_language">Taal</string>
+    <string name="title_settings_notification">Meldingen</string>
+    <string name="title_settings_sources">Bronnen</string>
+    <string name="title_settings_rate_app">Beoordeel App</string>
+    <string name="about_text">Met Happy News brengen wij jou de positiviteit die jij nodig hebt! Uit onze grote hoeveelheid bronnen halen wij alleen het meest positieve nieuws om jouw dag beter te maken. Je hoeft niet in te logggen, het is compleet gratis en 24/7 beschikbaar! \n\nHappy News heeft:\n - Een constante stroom aan positieve artikelen\n - Vrolijke tweets zorgvuldig door ons gefilterd\n - Inspirerende quotes om jou te motiveren\n\nWij van Happy news hebben als missie om iedereens dag wat positiever te maken. Wij maken een lange werkdag draaglijker door wat vrolijkheid en positiviteit te leveren wanneer jij het nodig hebt!</string>
```

Comment:
Fixed.

@DaanTul We should probably fix this in the Play Store listing as well.